versioned_hdf5/api.py

```diff
@@ -91,6 +111,23 @@ def current_version(self):
         """
         return self._versions.attrs['current_version']
 
+    @property
```

Comment:
We can create a setter property here too.

Reply:
:+1: